### PR TITLE
[minor] Declaration request in status expired #2298

### PR DESCRIPTION
**Pull request by magiccookie**

### CODE REVIEW

File: apps/ehealth/lib/ehealth/declaration_requests/declaration_requests.ex [hunk 1]
```diff
@@ -146,10 +146,20 @@ defmodule EHealth.DeclarationRequests do
...
-    DeclarationRequest
-    |> where([dr], dr.id == ^id)
-    |> filter_by_legal_entity_id(params)
-    |> Repo.one!()
+    p =
+      DeclarationRequest
+      |> where([dr], dr.id == ^id)
+      |> Repo.one!()
+
+    if Enum.empty?(p.data) do
```
Comment (Troush) [inline apps/ehealth/lib/ehealth/declaration_requests/declaration_requests.ex]: If you ever want to write a nested `if` you already doing somethign wrong
Just saing

Comment (magiccookie) [reply apps/ehealth/lib/ehealth/declaration_requests/declaration_requests.ex]: fixed.  if --> cond

File: apps/ehealth/lib/ehealth/declaration_requests/declaration_requests.ex [hunk 2]
```diff
@@ -145,13 +145,29 @@ defmodule EHealth.DeclarationRequests do
...
+  def get_by_id!(id, _) do
     DeclarationRequest
     |> where([dr], dr.id == ^id)
-    |> filter_by_legal_entity_id(params)
-    |> Repo.one!()
+    |> Repo.one()
   end
 
+  defp validate_data_legal_entity(%DeclarationRequest{data: data} = declaration_request, legal_entity_id)
+       when is_map(data) do
```
Comment (AlexKovalevych) [inline apps/ehealth/lib/ehealth/declaration_requests/declaration_requests.ex]: please remove guard

**Comment by lymychp** [comment]
@AlexKovalevych please roll back this PR

**Pull request merged**